images/linux/scripts/installers/android.sh

```diff
@@ -77,3 +78,5 @@ echo "y" | ${ANDROID_SDK_ROOT}/tools/bin/sdkmanager ${components[@]}
 
 # Add required permissions
 chmod -R a+rwx ${ANDROID_SDK_ROOT}
+
+invoke_tests "Android" "Android"
```

Comment:
@maxim-lobanov does it make sense to update `invoke_tests` to also accept single parameter with file name and run all tests in this file? Like we do for Windows Android https://github.com/actions/virtual-environments/blob/main/images/win/scripts/Installers/Update-AndroidSDK.ps1#L90

Reply:
Yes, agree.
I even think that it should work right now.
```
invoke_tests() {
    local TEST_FILE="$1"
    local TEST_NAME="$2"

    pwsh -Command "Import-Module '$HELPER_SCRIPTS/Tests.Helpers.psm1' -DisableNameChecking
        Invoke-PesterTests -TestFile \"$TEST_FILE\" -TestName \"$TEST_NAME\""
}
```
If the second param is not found, it will pass empty $TEST_NAME to `Invoke-PesterTests` and it is acceptable for it

Reply:
>If the second param is not found, it will pass empty $TEST_NAME to Invoke-PesterTests and it is acceptable for it

I had a error with empty test name: https://github.visualstudio.com/virtual-environments/_build/results?buildId=93134&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=5431112d-2b61-5a5f-7042-ef698f761043&l=71341

Reply:
You have outdated branch. Try to pull latest main

Reply:
It was merged yesterday: https://github.com/actions/virtual-environments/commit/90ecf07404f79b55f2f6aed4aac015cdb4706c78#diff-69b875f1d4923e904708eb39a8b3f7de8ba5f804170920a860b53da99a86d7caR12

Reply:
Removed test name.